plugins/grpc/cpp/v1.57.0/buf.plugin.yaml

```diff
@@ -0,0 +1,11 @@
+version: v1
+name: buf.build/grpc/cpp
+plugin_version: v1.57.0
+source_url: https://github.com/grpc/grpc
+description: Generates C++ client and server stubs for the gRPC framework.
+deps:
+  - plugin: buf.build/protocolbuffers/cpp:v24.0
```

Comment:
- Added in https://github.com/bufbuild/plugins/pull/714 
- Released in https://github.com/bufbuild/plugins/actions/runs/5870333103